drivers/modem/Kconfig.gsm

```diff
@@ -53,6 +53,14 @@ config MODEM_GSM_APN
 	  Specify Access Point Name, i.e. the name to identify Internet IP
 	  GPRS cellular data context.
 
+config MODEM_GSM_ATTACH_TIMEOUT
+	int "Number of packet service attach retries"
```

Comment:
Hmm, the prompt talks about retries, but the option name and the help text talks about time (seconds). This looks confusing now, perhaps the prompt should talk about time too.